packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -817,7 +817,13 @@ RichText.defaultProps = {
 
 const RichTextContainer = compose( [
 	withInstanceId,
-	withBlockEditContext( ( { clientId } ) => ( { clientId } ) ),
+	withBlockEditContext( ( { clientId, onFocus, isSelected }, ownProps ) => {
+		return { 
+			clientId: clientId,
+			isSelected: isSelected,
+			onFocus: onFocus || ownProps.onFocus
```

Comment:
This is needed for List Block to get the selection state. Since it's a Multiplatform block, the onFocus prop is not set directly.